tests/js/spec/views/performance/landing/widgets/widgetContainer.spec.jsx

```diff
@@ -216,6 +221,44 @@ describe('Performance > Widgets > WidgetContainer', function () {
     );
   });
 
+  it('Errors in widgets call PageError Provider', async function () {
+    const data = initializeData();
+
+    eventStatsMock = MockApiClient.addMockResponse({
+      method: 'GET',
+      url: `/organizations/org-slug/events-stats/`,
+      statusCode: 400,
+      body: {
+        detail: 'Request did not work :(',
+      },
+    });
+
+    wrapper = mountWithTheme(
```

Comment:
Would we be able to write this test using RTL instead of Enzyme? This one shouldn't require much to change in order to use RTL

Reply:
Good call on the RTL, but I'm going to probably convert the whole file once I come around and delete all the metrics code / test cases here (sometime this week or next I think), so I'll probably just leave it till then.

Reply:
Sounds good!